### PR TITLE
DIGI_12_GETALLTAXPARAMETERS

### PR DESCRIPTION
**Pull request by prakashkolli**
1.Added  TaxParameterDataModel for getting all the Tax Parameters from database

2.Added  TaxParameterController

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            try
+            {
+                taxParameterDataModel = new TaxParameterDataModel();
+                dtRecords = taxParameterDataModel.getAllTaxParameters();
+                dtRecords.TableName = "GETALLTAXPARAMETERS";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.NotFound, "No Record Found");
+            }
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs:36]: Change To internal server

File: DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+{
+    public class TaxParameterDataModel
+    {
+
+        //Author: Prakash kolli
+        //Date: 7-01-2020
+        //Description: To get all avaialble Tax Parameters from DataBase
+        //Story Id : DIGI 12
+        //Reviewer:
+        //Review Version:01
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs:15]: Remove Comments

File: DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs (hunk 1 above)
Comment (prakashkolli) [reply DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs:36]: changed to InternalServer status code

File: DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs (hunk 2 above)
Comment (prakashkolli) [reply DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs]: removed comments

File: DigiOrderApi/DigiOrderApi/DigiOrderApi.csproj [hunk 3]
```diff
@@ -59,8 +59,9 @@
     <Reference Include="Microsoft.Owin.Security.OAuth, Version=4.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.Owin.Security.OAuth.4.1.0\lib\net45\Microsoft.Owin.Security.OAuth.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.12.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\packages\Newtonsoft.Json.12.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
```
Comment (RAMUNIMMALA) [inline DigiOrderApi/DigiOrderApi/DigiOrderApi.csproj:64]: Packages should be always with latest versions

Comment (prakashkolli) [reply DigiOrderApi/DigiOrderApi/DigiOrderApi.csproj]: changed package version to latest version

File: DigiOrderApi/DigiOrderApi/DigiOrderApi.csproj [hunk 4]
```diff
@@ -59,7 +59,7 @@
     <Reference Include="Microsoft.Owin.Security.OAuth, Version=4.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.Owin.Security.OAuth.4.1.0\lib\net45\Microsoft.Owin.Security.OAuth.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+     <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/DigiOrderApi.csproj:62]: Remove space

File: DigiOrderApi/DigiOrderApiDataAccess/DigiOrderApiDataAccess.csproj [hunk 5]
```diff
@@ -30,6 +30,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\packages\Newtonsoft.Json.12.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/DigiOrderApiDataAccess.csproj:36]: Remove these packages

**Pull request merged**